### PR TITLE
Updates to NCI pages and notes on `parallel` in computations

### PR DESCRIPTION
**Pull request by hot007**
Addresses #40, #67 and maybe #51

### CODE REVIEW

**Review by paolap — COMMENTED** [review]
In the gadi/ARE file you're mentioning:
"though usually you would start exploratory data work on the NCI's OOD (see below) " I would use ARE here instead as while we might not want to remove the OOD page yet, we should still stir away users from OOD.

Similarly I found confusing having OOD again mentioned here, as if the assumption is that we're talking about OOD, as this the ARE page:
"If you need to work interactively using code in your Gadi `/home` directory instead of the [OOD](https://acdguide.github.io/BigData/platforms/platforms-nci-ood.html), the ARE provides a JupyterLab or VDI environment with access to Gadi `/home` and is the recommended tool for users looking to perform interactive large-scale data analysis."
Finally I would put this:
"There are also [helper scripts](https://github.com/coecms/nci_scripts) such as `gadi_jupyter` provided by the CLEX CMS team though these are largely deprecated in favour of the ARE or OOD. "
separately as a
```{warning}
```
and changing the text to If you are still using ... be aware it is deprecated in favour of ARE"
AS we don't want to tell them of a deprecated option unless they are already using it.
 


Last sentence "Note that ARE usage ..." you can use a Note box
```{note}
your text
```

**Comment by paolap** [comment]
Thanks @hot007 :-)

**Comment by hot007** [comment]
Okay, this one should be good to go pending @paigem 's review (or anyone else who feels like it!?). Thanks! I learned a new thing today :)

File: BigData/platforms/platforms-nci-gadi.md [hunk 1]
```diff
@@ -23,12 +23,15 @@ Launching the VDI via the button gives access to a familiar linux graphical inte
 
 ## When would I use this system? 
 
-- Gadi is intended for large scale modelling and data processing jobs. If you are running parallel numerical models at scale then you would likely work exclusively on Gadi or a similar system. It is also useful for "high throughput computing" like AI/ML workflows, and "high performance data" parallel data processing using Pangeo tools like python's `dask` library - though usually you would start exploratory data work on the NCI's OOD (see below) and only move to Gadi when greater scalability is required.
+- Gadi is intended for large scale modelling and data processing jobs. If you are running parallel numerical models at scale then you would likely work exclusively on Gadi or a similar system. It is also useful for "high throughput computing" like AI/ML workflows, and "high performance data" parallel data processing using Pangeo tools like python's `dask` library - though usually you would start exploratory data work on the NCI's virtutal desktop or jupyterLab via the ARE and move to Gadi batch jobs when greater scalability is required.
```
Comment (paigem) [inline BigData/platforms/platforms-nci-gadi.md:26]:
```suggestion
- Gadi is intended for large scale modelling and data processing jobs. If you are running parallel numerical models at scale then you would likely work exclusively on Gadi or a similar system. It is also useful for "high throughput computing" like AI/ML workflows, and "high performance data" parallel data processing using Pangeo tools like python's `dask` library - though usually you would start exploratory data work on the NCI's virtual desktop or JupyterLab via the ARE and move to Gadi batch jobs when greater scalability is required.

```

File: BigData/platforms/platforms-nci-ood.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 (OOD)=
 # NCI OOD Virtual Desktop Infrastructure
 
-**NOTE that NCI have recently launced their ARE platform, which may ultimately replace this OOD offering, this information is current as of July 2022.**
+**NOTE that NCI have recently launced their ARE platform, which may ultimately replace this OOD offering, this information is current as of November 2022.**
```
Comment (paigem) [inline BigData/platforms/platforms-nci-ood.md:4]:
```suggestion
**NOTE that NCI have recently launched their ARE platform, which may ultimately replace this OOD offering (this information is current as of November 2022).**

```

File: BigData/computations/computations.ipynb [hunk 3]
```diff
@@ -104,15 +104,21 @@
...
+    "* [xargs](https://en.wikipedia.org/wiki/Xargs) - some people prefer `xargs` to `parallel`, it can be used in conjunction with commands like `find` to permit operations simultaneously on many files.\n",
     "\n",
+    "{ref}`cdo` and {ref}`nco` are powerful command line tools built around netCDF data.\n",
     "You can also use multiple parallel threads in CDO with <br>\n",
     "    `cdo -P <nthreads> ...`  <br>\n",
     "and in NCO with <br>\n",
     "    `ncks --thr_nbr <nthreads> ...`\n",
-    " \n",
+    "\n",
+    "Python `multiprocessing.Pool.map` can also be used when working in a python environment and parallel behaviour is needed (not a sensible alternative to `dask` for data processing).\n",
```
Comment (paigem) [inline BigData/computations/computations.ipynb:120]: Could we add another few words to explain why `multiprocessing` is is not a good alternative to `Dask` for data processing? I don't know much about multiprocessing myself, so will defer to others @hot007 @paolap.

**Review by paigem — APPROVED** [review]
Looks great! Just a couple minor comments. Thanks @hot007!!

**Review by paolap — APPROVED** [review]

File: BigData/computations/computations.ipynb (hunk 3 above)
Comment (paolap) [reply BigData/computations/computations.ipynb:120]: Dask is easier to use as it figures out on its own how to implement the parallelisation and can be scaled to a cluster.

Comment (paigem) [reply BigData/computations/computations.ipynb:120]:
```suggestion
    "Python `multiprocessing.Pool.map` can also be used when working in a python environment and parallel behaviour is needed. Dask is typically easier to use for parallel data analysis, as it figures out on its own how to implement the parallelisation and can also be scaled to a cluster.\n",
```

**Review by paigem — COMMENTED** [review]
Added suggested wording about Dask and multiprocessing.

**Comment by paolap** [comment]
I think this is nearly ready to be merged, just waiting @hot007 to review Paige's suggestions :-)

**Comment by hot007** [comment]
Accepted suggestions, thanks heaps for the critical review both, and @paigem for the nice suggestions. Will merge now.

**Pull request merged**